clients/cli/build/update_brew.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -eo pipefail
+
+brew_dir=$(mktemp -d -t ci-XXXXXXXXXX)
+git clone --depth 1 https://$API_TOKEN_GITHUB@github.com/phrase/homebrew-brewed.git $brew_dir &> /dev/null
```

Comment:
I would probably now swallow all output here as there could be plenty of reasons why the cloning could not work.

Maybe write the output some where tmp and then cat if the command fails?